meeting-notes/2018/2018-11-19--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,221 @@
+      - http://www.news.cs.nyu.edu/~jinyang/pub/phd.pdf
+    - Sync call with @jacobheun and @bigs
+  - Lot's of reviews for libp2p@0.24.0
+  - Fix issue with pin cli in js-ipfs
+    - [js-ipfs#1719](https://github.com/ipfs/js-ipfs/pull/1719)
+- Blocked: N/A
+- Next:
+  - `service-worker-gateway` issues
+    - [js.ipfs.io#207](https://github.com/ipfs/js.ipfs.io/issues/207)
+  - IPNS over DHT PR
+  - IPNS over Pubsub test Interop using go and js fixes
+  - Stress tests / Testbed DHT
+
+@alanshaw
+- Done:
+    - Reviewed and merged files API refactor for [interface-ipfs-core](https://github.com/ipfs/interface-ipfs-core/pull/378) & [js-ipfs-api](https://github.com/ipfs/js-ipfs-api/pull/878)
+    - Reviewed and merged ipld-dag-pb changes for [interface-ipfs-core](https://github.com/ipfs/interface-ipfs-core/pull/388) & [js-ipfs-api](https://github.com/ipfs/js-ipfs-api/pull/889)
+    - Reviewed and made merge ready [files API refactor and ipld-dag-pb changes](https://github.com/ipfs/js-ipfs/pull/1720) for js-ipfs
+    - Added HTTP API tests to [`cid-base` PR](https://github.com/ipfs/js-ipfs/pull/1552) (still core tests TODO)
+    - Reviewed [IPNS routing logic refactor PR](https://github.com/ipfs/js-ipfs/pull/1701)
+    - Re-enabled ability to [lazy load IPLD formats](https://github.com/ipfs/js-ipfs/pull/1704) in js-ipfs
+    - Began planning for [0.34 release](https://github.com/ipfs/js-ipfs/issues/1721)
+    - Pair programming with [@pkafei](https://github.com/pkafei)
+- Blocked:
+    - [Object API breaking change to return CIDs](https://github.com/ipfs/interface-ipfs-core/pull/388#issuecomment-437866965) "soft blocks" a release of js-ipfs-api.
+- Next:
+    - Implement `addFrom*` methods in js-ipfs
+    - Add core interface tests for [`cidBase` option PR](https://github.com/ipfs/js-ipfs/pull/1552)
+    - Write interop tests for CIDv0/1 put and get
+
+
+@hugomrdias
+- Done:
+    - fix ctl options and normalize behavior between the client, daemon and in-proc
+        - Fix non disposable daemon init/start and attach to running daemons [#308](https://github.com/ipfs/js-ipfsd-ctl/pull/308)
+    - fix: add missing dependencies [#1663](https://github.com/ipfs/js-ipfs/pull/1663)
+    - Fails to init with Node.js 11.x - Lock FcntlFlock failed: EINVAL, Invalid argument [#1700](https://github.com/ipfs/js-ipfs/issues/1700)
+        - fix: fix lock for node 11 [#181](https://github.com/ipfs/js-ipfs-repo/pull/181)
+    - fix: remove electron-webrtc and wrtc for now [#1718](https://github.com/ipfs/js-ipfs/pull/1718)
+    - [WIP] fix: remove dexie and unused deps [#309](https://github.com/ipfs/js-ipfsd-ctl/pull/309)
+    - chore: add renovate [#1722](https://github.com/ipfs/js-ipfs/pull/1722)
+    - update gitlab ci config and added docs according to the feedback
+    - started working on libp2p-mplex async iterators
+- Blocked
+    - fix: pass config to init [#303](https://github.com/ipfs/js-ipfsd-ctl/pull/303)
```

Comment:
@hugomrdias lets open an issue for go-ipfs and add comments to the code in this PR for referencing it.